lib/Command/Add.php

```diff
@@ -393,17 +375,10 @@ protected function askUnsignedInt(string $subject, string $label, InputInterface
 		$input->setOption($subject, $helper->ask($input, $output, $q));
 	}
 
-	protected function stringNormalizer($input) {
+	protected function stringNormalizer(?string $input): string {
 		return $input ? trim($input) : '';
 	}
```

Comment:
Normalizes '0' into '' most likely, not sure if it is a big deal.

Reply:
Why would it?

Reply:
'0' is falsy: https://www.php.net/manual/en/language.types.boolean.php#language.types.boolean.casting
Something like this would fix it:
```suggestion
	protected function stringNormalizer(?string $input): string {
		return is_string($input) ? trim($input) : '';
	}
```
Or 
```suggestion
	protected function stringNormalizer(?string $input): string {
		return ($input !== null) ? trim($input) : '';
	}
```